textattack/shared/utils/tensor.py

```diff
@@ -1,119 +1,31 @@
+import numpy as np
 import torch
 
-import textattack
 
-# from textattack.shared import utils
+def batch_model_predict(model_predict, inputs, batch_size=32):
```

Comment:
Wow this seems much nicer now. I assume it still works same as before?

Reply:
yep (all the tests pass anyway)